Dockerfile.ubi

```diff
@@ -0,0 +1,233 @@
+ARG PYTHON_VERSION
+
+FROM nvcr.io/nvidia/cuda:12.8.1-devel-ubi9
+
+ARG PYTHON_VERSION
+
+WORKDIR /workspace
+
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    UV_LINK_MODE=copy \
+    TORCH_CUDA_ARCH_LIST="9.0;10.0+PTX" \
+    PYTHON_VERSION=${PYTHON_VERSION} \
```

Comment:
```suggestion
FROM nvcr.io/nvidia/cuda:12.8.1-devel-ubi9

ARG PYTHON_VERSION

WORKDIR /workspace

ENV LANG=C.UTF-8 \
    LC_ALL=C.UTF-8 \
    UV_LINK_MODE=copy \
    TORCH_CUDA_ARCH_LIST="9.0;10.0+PTX" \
    PYTHON_VERSION=${PYTHON_VERSION:-3.12} \
```

Reply:
You proposed fix looks good, ARGS above FROM is only useful if we need the vars in the FROM declaration, we don't so can drop. +1 on a default of 3.12